src/api/babel.config.js

```diff
@@ -11,10 +11,5 @@ module.exports = {
     ],
     '@babel/preset-typescript',
   ],
-  plugins: [
-    '@babel/plugin-transform-runtime',
-    '@babel/plugin-proposal-class-properties',
-    '@babel/proposal-object-rest-spread',
-    '@babel/plugin-syntax-dynamic-import',
-  ],
+  plugins: ['@babel/plugin-transform-runtime', '@babel/plugin-proposal-class-properties', '@babel/proposal-object-rest-spread'],
```

Comment:
Note for any reviews/historic investigations - `'@babel/plugin-syntax-dynamic-import'` is included by default now.